test/functionalTest/cases/1048_subscription_cache/csub_update_with_notif.test

```diff
@@ -274,7 +274,7 @@ Date: REGEX(.*)
         },
         "lastNotification": "REGEX(.*)",
         "lastSuccess": "REGEX(.*)",
-        "timesSent": 2
+        "timesSent": REGEX((1|2))
```

Comment:
I think the problem was due to this part:

```
brokerStart CB 0 IPv4 -subCacheIval 2
...

--SHELL--

#
# Note that we use sleep in three places to give the DB time to sync with regards to
# the cache (WAIT_TIME = 1.5 seconds)
#
# Under valgrind we need to sleep a little longer (2 seconds)
#
WAIT_TIME=1.5s
if [ "$FUNC_TEST_RUNNING_UNDER_VALGRIND" == "true" ]
then
  WAIT_TIME=2s
fi
```

Originally subCacheIval was 1. It was increased to 2 due to valgrind but it seems the WAIT_TIME was not updated consecuencly. It has to be greater that sub cache interval, otherwise CB will not sync as expected.

Thus, let's adjust times, for instance:

```
WAIT_TIME=2.5s
if [ "$FUNC_TEST_RUNNING_UNDER_VALGRIND" == "true" ]
then
  WAIT_TIME=3s
fi
```

I think is a better solution than REGEX(1|3), as it is more deterministic.

Reply:
OK. I can try.
If it works both as normal functest and under valgrind we use that solution.
What value should I expect for  "timesSent" ?

Reply:
Took some time to actually look at the test.
The change makes a lot of sense.

Fixed in de07e75